app/views/submissions/_available_evaluators.html.erb

```diff
@@ -0,0 +1,44 @@
+<% @available_evaluators = available_evaluators(@submission) %>
+<div class="usa-accordion">
+  <% rand = SecureRandom.hex(8) %>
+  <h2 class="usa-accordion__heading" id="<%= "#{rand}-accordion-heading" %>">
+    <button
+      type="button"
+      class="usa-accordion__button display-flex flex-justify icon-white"
+      aria-expanded="true"
+      aria-controls="<%= rand %>"
+    >
+      <div>Available Evaluators</div>
+    </button>
+  </h2>
+
+  <div class="usa-accordion__content background-primary-lighter" id="<%= rand %>">
+    <% unless @available_evaluators.empty? %>
+      <% @available_evaluators.each do |evaluator| %>
+              <div class="float-left">
+                  <%= "#{evaluator.first_name} #{evaluator.last_name}" %> <br/>
+                  <%= evaluator.email %>
+              </div> 
+              <div class="float-right">
+                  <% assignment = evaluator.evaluator_submission_assignments.where(submission_id: @submission.id).first %>
+                  <% if assignment %>
+                    <%= form_with url: phase_evaluator_submission_assignment_path(@submission.phase, assignment.id), method: "PATCH" do |f| %>
+                      <%= f.hidden_field :phase_id, value: @submission.phase.id %>
+                      <%= f.hidden_field :status, value: "assigned" %>
+                      <%= f.submit "Reassign", class: "usa-button font-body-2xs" %>
+                    <% end %>
+                  <% else %>
+                    <%= form_with url: phase_evaluator_submission_assignments_path(@submission.phase) do |f| %>
+                      <%= f.hidden_field :evaluator_id, value: evaluator.id %>
+                      <%= f.hidden_field :submission_id, value: @submission.id %>
+                      <%= f.submit "Assign", class: "usa-button font-body-2xs" %>
+                    <% end %>
+                  <% end %>  
+              </div>
+
+      <% end %>
+    <% else %>
+      No available evaluators.
```

Comment:
There was an update to the empty message here and some styling..
<img width="348" alt="Screenshot 2025-01-07 at 11 00 00 AM" src="https://github.com/user-attachments/assets/deffd43a-ca37-4668-920a-a6034d820c8d" />